docs/content/patterns/yourgov-example-application/index.mdx

```diff
@@ -0,0 +1,247 @@
+
+It’s important to move focus to the title of the current step as the user navigates through the form. We do this so users can start interacting straight away.
+
+#### Single-page form
+
+This demonstration of the [Single-page form](/templates/single-page-form) pattern multi-question form. It illustrates how an invitation can be sent to add someone as an employee within an application.
+
+<ResponsivePreview
+	href="/yourgov/app/staff/invite-staff-member"
+	label="Single-page form"
+	padding={false}
+	title="yourGov: single-page form preview"
+	referrerLabel={title}
+/>
+
+#### Multi-page form
+
+yourGov includes a [Multi-page form](/templates/multi-page-form) pattern that breaks up a large form into multiple smaller steps. In this example, a food truck permit application has been used.
+
+Within this form, you will see some patterns that are typical of a form flow:
+
+- [Getting started page](#form-getting-started-page)
+- Presenting data that is already known by the system (Tell Us Once)
+- Editing data that is already known by the system
+- Conditional reveal of relation question/s
+- Address input
+- Grouped fields
+- Add records to an empty state
+- Upload a defined list of files to a table
+- Entry of a date and date range.
+- Review and submit
+- Success page
+
+<ResponsivePreview
+	href="/yourgov/app/permits/apply-for-new-permit/mobile-food-vendor-permit"
+	label="Multi-page form"
+	padding={false}
+	title="yourGov: single-page form preview"
+	referrerLabel={title}
+/>
+
+<h4 id="form-getting-started-page">Form “Getting started” page</h4>
+
+A Getting started page helps users understand what to expect before beginning a multi-page form. It typically includes key information such as:
+
+- what the form is for and how it works
+- what the user needs to complete the form (e.g. documents or account details)
+- how long the form usually takes to complete
+- how to get help or find further information.
+
+It may also outline what will happen to the user’s information, any consent they’ll need to give, and what will happen after submission.
+
+In some cases, a question may be included on the Getting started page to tailor the form experience based on the user’s situation. In this case, it is the type of mobile food vendor permit being requested.
+
+<ResponsivePreview
+	href="/yourgov/app/permits/apply-for-new-permit/mobile-food-vendor-permit"
+	label="Form “Getting started” page"
+	padding={false}
+	title="yourGov: “Getting started” page preview"
+	referrerLabel={title}
+/>
+
+<BackToTop />
+
+### Managing datasets
+
+#### Complex data table
+
+yourGov includes a complex data table that displays staff records. The table supports advanced interactions such as sorting, [filtering](/patterns/search-filters), individual and batch actions. Users are able to see, compare, and manage a dataset.
```

Comment:
I think we need to clarify "individual" with "actions". When you read it, you don't know what it applies to
```suggestion
yourGov includes a complex data table that displays staff records. The table supports advanced interactions such as sorting, [filtering](/patterns/search-filters), individual actions and batch actions. Users are able to see, compare, and manage a dataset.
```